src/PowerShellEditorServices/Services/Symbols/PsdDocumentSymbolProvider.cs

```diff
@@ -15,9 +15,11 @@ namespace Microsoft.PowerShell.EditorServices.Services.Symbols
     /// Provides an IDocumentSymbolProvider implementation for
     /// enumerating symbols in .psd1 files.
     /// </summary>
-    public class PsdDocumentSymbolProvider : IDocumentSymbolProvider
+    internal class PsdDocumentSymbolProvider : IDocumentSymbolProvider
     {
-        IEnumerable<SymbolReference> IDocumentSymbolProvider.ProvideDocumentSymbols(
+        string IDocumentSymbolProvider.ProviderId => nameof(PsdDocumentSymbolProvider);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private property 'IDocumentSymbolProvider.ProviderId'.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4979332)